tests/roles/backend_services/tasks/main.yaml

```diff
@@ -104,8 +104,8 @@
     oc get pod openstack-galera-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
   register: mariadb_running_result
   until: mariadb_running_result is success
-  retries: 60
-  delay: 2
+  retries: 30
+  delay: 10
```

Comment:
10 seconds seems like an unnecessarily long time to wait between the check attempts? The CLI call is fairly cheap, we can run it many times. Not a blocker but can we just increase the number of retries with delay staying 2 secs?

Reply:
That's fine. Fixed in next PR update